openapi.yaml

```diff
@@ -0,0 +1,319 @@
+openapi: 3.0.3
+info:
+  title: Todo App
+  description: |-
+    This is a todo application back-end that follows the REST API standard and OpenAPI 3.0 specification and provides full functionality for managing tasks.
+  contact:
+    email: satikajayawardena@proton.me
+  version: 1.0.0
+externalDocs:
+  description: The Todo App repository
+  url: https://github.com/satikaj/RESTful-app
+servers:
+  - url: http://localhost:3000
+tags:
+  - name: todo
+    description: Manage todo items
+  - name: user
+    description: Manage user account
+paths:
+  /todo:
+    get:
+      tags:
+        - todo
+      summary: Get all todos
+      description: Get all todos of the current user
+      operationId: getAllTodos
+      responses:
+        '200':
```

Comment:
Success response is good but not enough. Thinks about the consumers of this API documentation when designing an API.
Possible consumers are

1. Business analyst of the domain.
2. Product owner
3. Mobile app developers
4. Web application developers

So if this documents only provides the success case  they would not understand how to deal with the unsuccessful cases.
Including 400 Bad Request, 500 Internal Server Error , unsuccessful authentication attempt, a slow system etc.